nrcemt/alignment_software/gui/optimization/step_optimization.py

```diff
@@ -126,29 +126,34 @@ def perform_optimization(self):
             # find x, y, z locations for each particle
             markers = self.marker_data
             normalized_markers = normalize_marker_data(markers)
-            x, y, z, alpha, phai = optimize_particle_model(
+            x, y, z, alpha, phai, acc = optimize_particle_model(
```

Comment:
is acc supposed to be short for accuracy?

Reply:
yeah, I can change it to have the full word